package.json

```diff
@@ -54,7 +54,7 @@
     "@aws-sdk/client-dynamodb": "^3.624.0",
     "@aws-sdk/client-secrets-manager": "^3.624.0",
     "@aws-sdk/util-dynamodb": "^3.624.0",
-    "@azure/msal-node": "^2.16.1",
```

Comment:
Why are we updating it? I'm fine with it but is there a specific reason?

Reply:
There is no specific reason the extension do not use azure. Will revert.